src/js/components/sharedComponents/commonCards/CardButton.jsx

```diff
@@ -24,37 +25,49 @@ const propTypes = {
 };
 
 const CardButton = ({
-    link, govLink, action, text, variant = "secondary", customClassName = '', children
+    link, govLink, onlyPerformAction = "false", action, text, variant = "secondary", customClassName = '', children
 }) => {
     const variantMapper = {
         primary: "card__button--primary",
         secondary: "",
         text: "card__button--borderless"
     };
 
+    if (!onlyPerformAction) {
```

Comment:
This condition isn't working for some reason.  All of the links on the page are now buttons.  The layout is throw off a bit for the links now.  
![Screen Shot 2022-09-30 at 4 05 25 PM](https://user-images.githubusercontent.com/121010/193349913-253f0333-b788-4a5a-86e9-7461e104a336.png)

To avoid too many conditions and props here, could these be changed to links here and styled as buttons since they basically function like a link and redirect the user to a new location?  or the condition could be based on the text variant.  Ie. if the variant is text use the link element else use the button element.

Reply:
fysa the reason i added the condition to only an action is because the card button would direct to the search page and then the page would update bc of the action so it was loading the page twice

Reply:
not sure what i want to do to fix it yet

Reply:
switched the condition around so it's less confusing